wlauto/result_processors/cpustate.py

```diff
@@ -163,8 +163,10 @@ def process_iteration_result(self, result, context):
         self.logger.debug('Generating power state reports from trace...')
         if self.create_timeline:
             timeline_csv_file = os.path.join(context.output_directory, 'power_states.csv')
+            cpu_utilisation = os.path.join(context.output_directory, 'cpu_utilisation.csv')
```

Comment:
for now leave existing prameter `create_timeline` for compatibility and the new parameter should be `create_utilization_timeline`. The difference between them should be made clear with the descriptions (updating `create_timelines`'s if need be too).

In WA v3 which we are currently working on we can add aliases to parameters and we will rename original parameter to `create_powerstate_timeline`, aliasing it as `create_timeline`